data_steward/tools/retract_data_bq.py

```diff
@@ -615,24 +481,27 @@ def int_list_to_bq(l):
     return "(%s)" % ', '.join(str_l)
 
 
-def run_retraction(project_id, pids, hpo_id, deid_flag=False):
+def run_retraction(project_id, sandbox_dataset_id, pid_table_id, hpo_id, dataset_ids=None):
     """
     Main function to perform retraction
```

Comment:
Please describe in docstring
1. the expected pid table schema and the meaning of the columns
1. what is deleted when this is run
1. potential errors or exceptions raised

Reply:
I've added this for both the bq and gcs `run_retraction` functions, and allowed gcs retraction to fail if certain exceptions are encountered.